src/index.html

```diff
@@ -11,6 +11,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <section class="search">
+      <form
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <div class="search__block search__block-big">
+          <img
+            class="img"
+            src="images/Search.svg"
+            alt="Search"
+          >
```

Comment:
Move the image inside the label. So in this case you don't need to additional div wrapper here